Cargo.toml

```diff
@@ -33,8 +33,8 @@ ron = "0.10"
 thiserror = "*"
 glam = { version = "0.29" }
 uuid = { version = "1.13"}
-yaserde = "0.7"
-sdformat_rs = { git = "https://github.com/open-rmf/sdf_rust_experimental", rev = "9fc35f2"}
+yaserde = "0.12"
+sdformat_rs = { git = "https://github.com/open-rmf/sdf_rust_experimental", branch = "luca/experimental_params_new_yaserde" }
```

Comment:
This is probably me being "Kay-Poh" but it might be worth also simply re-exporting `ya-serde` upstream inside `sdformat_rs` so we don't need to explicitly import it here.

Reply:
That's a good call, https://github.com/open-rmf/rmf_site/pull/319/commits/da7a35a05f63707965b01381702f3a8a5ee91f14